### ISSUE TITLE
Unable to delete hot-topic

### ISSUE THREAD

**Issue by nadavkav** [hit-moodle/moodle-mod_hotquestion#20]
As a teacher, I would like to remove some topics from the list and I am unable to.
Also as a student, I would like to remove one of my posts (topics) and I am unable to.

**Comment by nadavkav** [comment]
Any news on this one?

**Comment by sunner** [comment]
Sorry for the really delayed reply.

Since I changed my job, I have no enough enthusiasm to maintenance this project. Anyone who would like to take over it is welcome.

**Comment by nadavkav** [comment]
Thank you sunner for your quick reply. I will see if I can fix it myself and upload a merge request?
(Or should I fork it?)

**Comment by sunner** [comment]
Yes, of coz you can. In my opinion, you'd better fork and redistribute it if you love to continue this project. Thanks in advance.
